velox/vector/arrow/Bridge.h

```diff
@@ -36,6 +36,7 @@ struct ArrowOptions {
   bool flattenDictionary{false};
   bool flattenConstant{false};
   TimestampUnit timestampUnit = TimestampUnit::kNano;
+  std::string timestampTimeZone = "";
```

Comment:
feels like this should be an optional value so we can differentiate empty string from not set?

Reply:
@pedroerp Yes. Updated to `std::optional<std::string>`.